mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
IIRC smugglers should use ordos(?) artwork for everything except their custom starport, which we currently have set up as corrino.